quasar-core/src/main/java/co/paralleluniverse/fibers/instrument/JavaAgent.java

```diff
@@ -183,6 +183,12 @@ public void error(String msg, Throwable exc) {
             }
         });
 
+        // CORDA-3666: Access Classes now so we don't deadlock while
+        // loading it later.
+        // We expect this call to isYieldMethod to return true.
+        // Prints "Classes ready: true"
+        instrumentor.log(LogLevel.DEBUG, "Classes ready: %s", Classes.isYieldMethod(Classes.FIBER_CLASS_NAME, "park"));
```

Comment:
This is slightly non-obvious. Am I correct in thinking that there is nothing special about invoking `Classes.isYieldMethod()`, because all you really need to happen here is for the JVM to execute
```java
Classes.<clinit>
```

Reply:
Yes. That's correct.

Reply:
I've added a comment explaining what we are up to.